README.md

```diff
@@ -288,17 +295,77 @@ or:
 
 For more information about Android demo and the complete list of available expressions go to [demo/ios/RhinoDemo](/demo/ios/RhinoDemo).
 
-### JavaScript Demos
+### Web Demos
 
-From [demo/javascript](/demo/javascript) run the following in the terminal:
+From [demo/web](/demo/web) run the following in the terminal:
 
-```bash
+```yarn
+yarn
+yarn start
+```
+
+(or)
+
+```
+npm install
+npm run start
+```
+
+Open http://localhost:5000 in your browser to try the demo.
```

Comment:
we should show the YAML on the demo page. I think we can probably grab it programmatically from the binding?